pom.xml

```diff
@@ -6131,7 +6124,7 @@
                 <version>${version.org.wildfly.arquillian}</version>
                 <exclusions>
                     <exclusion>
-                        <groupId>org.wildfly.security</groupId>
+                        <groupId>org.wildfly.core</groupId>
```

Comment:
This is a different artifact -- it's the actual security manager, not the subsystem.